### ISSUE TITLE
mismatch description displays NaN as null when comparing objects

### ISSUE THREAD

**Issue by UrsMetz** [moll/js-must#56]
When comparing objects using `eql` and the actual object has a property that is `NaN` the diff of the failed assertion is misleading: the following assertion
```javascript
const expect = require("must")

expect({a: NaN}).to.eql({a: 1})
```
results in this message:
```
AssertionError: {"a":null} must be equivalent to {"a":1}
```

The root cause is that `json-stringify-safe` converts `NaN` to `null`.

**Comment by moll** [comment]
You're absolutely right! Thanks for bringing this to my attention again! I fixed that and released v0.13.3. Please give it a try and let me know if it wasn't what you were expecting! I'll close this until again. :)

**Comment by UrsMetz** [comment]
@moll just gave it a try and it works as expected, thanks for the fast fix and release! :-)
